build/configs/artik053/artik053_download.sh

```diff
@@ -119,6 +119,26 @@ main()
 			romfs
 			;;
 
+		OS|os)
+			echo "OS only :"
+
+			# check existence of os binary
+			if [ ! -f "${OUTPUT_BINARY_PATH}/tinyara_head.bin" ]; then
+				echo "TinyAra binary is not existed, build first"
+				finish_download 1
+			fi
+
+			# Download all binaries using openocd script
+			pushd ${OPENOCD_DIR_PATH}
+			${OPENOCD_BIN_PATH}/openocd -f artik053.cfg -c ' 	\
+			flash_write os ../../../../output/bin/tinyara_head.bin;	\
+			exit'
+			popd
+
+			# check romfs and download it
+			romfs
```

Comment:
You wrote "OS Only", but here is romfs. What is right?

Reply:
@sunghan-chang 'romfs' is a kind of user data, not binary. So i mean only os binary will be downloaded. If you want I will remove 'only' word.

Reply:
OS and rom are different partitions.  When you say "OS only", including rom is wrong. And os and rom don't have any dependancy. We don't need to update both together.